### ISSUE TITLE
[GUI] Consider removing blurred elipse from 5 main menu entries

### ISSUE THREAD

**Issue by pedropaulosuzuki** [supertuxkart/stk-code#5109]
### Description

In the main menu, the selected item already changes color, so it is a bit redundant to have the ellipse down there as well. And it makes the UI look a bit more dated.

Refering to the blurred white ellipse under "Single Player" (the selected box) on the screenshot below:

![Screenshot of Super Tux Kart 1.4 main menu](https://github.com/supertuxkart/stk-code/assets/32345702/607be8ac-dbb0-4c7a-afb3-85c1041eb15d)

Maybe it could be changed to look less dated, maybe like a glow around the box or something, but removing is fine, in theory.

Not talking about removing the glow in other areas other than the main menu, or even the ones in the bottom right corner, since they do not have the same behavior of changing colors when selected. Only talking about the 5 big icons in the center.

### Configuration
STK release version:  1.4 Flatpak

**Comment by CodingJellyfish** [comment]
Indeed, the icon buttons having a dedicated focus image should have a different logic from those don't.
